webview-ui/src/Kaito/KaitoModels.tsx

```diff
@@ -115,19 +122,21 @@ resource:
       apps: ${appname}
 inference:
   preset:
-    name: ${inferencename}`;
-        return yaml;
+    name: ${inferencename}${privateConfig}`;
+        return [yaml, gpu];
     }
 
     function generateCRD(model: string) {
-        const yaml = generateYAML(model);
+        const yaml = generateYAML(model)[0];
```

Comment:
💡 Quick note: What is the relevance of `generateYAML(model)[0]` what standard value `[0]` will have and add sample value as comment for cahnces if this contract gets violated in future at least we would know what was the extended valuer.